app/services/doc_auth/response.rb

```diff
@@ -77,5 +79,14 @@ def network_error?
     def selfie_check_performed?
       @selfie_check_performed
     end
+
+    def selfie_success
+      # to be implemented by concrete subclass
+    end
+
+    def doc_auth_success?
+      # to be implemented by concrete subclass
+      false
+    end
```

Comment:
Use dummy implementations, since Ruby does not have Interface, and it becomes cumbersome to add more and more properties.

Reply:
Would we want to `raise NotImplementedError` instead of the comment?

Reply:
@night-jellyfish ,that's what I thought first, but we use DocAuth::Response object in a lot of tests, and we call these two methods in __to_h__ method, in this case, all those tests will throw this exception.

Reply:
I see, thank you for clarifying!